bulldozer/config_v1.go

```diff
@@ -14,26 +14,7 @@
 
 package bulldozer
 
-type MessageStrategy string
-type TitleStrategy string
-type MergeMethod string
-
-const (
-	PullRequestBody  MessageStrategy = "pull_request_body"
-	SummarizeCommits MessageStrategy = "summarize_commits"
-	EmptyBody        MessageStrategy = "empty_body"
-
-	PullRequestTitle   TitleStrategy = "pull_request_title"
-	FirstCommitTitle   TitleStrategy = "first_commit_title"
-	GithubDefaultTitle TitleStrategy = "github_default"
-
-	MergeCommit     MergeMethod = "merge"
-	SquashAndMerge  MergeMethod = "squash"
-	RebaseAndMerge  MergeMethod = "rebase"
-	FastForwardOnly MergeMethod = "ff-only"
-)
-
-type MergeConfig struct {
+type MergeConfigV1 struct {
 	Whitelist Signals `yaml:"whitelist"`
```

Comment:
We could also rename this on code level and keep just the yaml annotation.. (same for blacklist and for update )